cmd/gitops/add/app/cmd.go

```diff
@@ -118,6 +118,15 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create app service: %w", err)
 	}
 
+	wegoConfig, err := kubeClient.GetWegoConfig(ctx, params.Namespace)
+	if err != nil {
+		return fmt.Errorf("failed getting wego config")
+	}
+
+	if params.ConfigRepo == "" {
+		params.ConfigRepo = wegoConfig.ConfigRepo
```

Comment:
Will this logic need to be repeated on API calls as well? Maybe this should be within the `.Add()` func?